gateway/policy.go

```diff
@@ -18,8 +18,8 @@ import (
 )
 
 type DBAccessDefinition struct {
-	APIName              string                       `json:"apiname"`
-	APIID                string                       `json:"apiid"`
+	APIName              string                       `json:"api_name"`
```

Comment:
I think this lacks bson too

Reply:
it's not been used in dashboard. 
dashboard maintains separate structs for this and that's root cause for this disconnect.